source/CCodeInjector.cpp

```diff
@@ -1,35 +1,40 @@
 #include "stdafx.h"
-#include "cleo.h"
+#include "CleoBase.h"
```

Comment:
This is already current state of the code. I do not know how you managed to create PR with such diff.